server/base/src/main/java/org/apache/accumulo/server/util/Admin.java

```diff
@@ -494,22 +494,29 @@ private static void flushAll(final ClientContext context) {
     try {
       flusher.join(3000);
     } catch (InterruptedException e) {
-      // ignore
+      log.warn("Interrupted while waiting to join Flush thread");
     }
 
     while (flusher.isAlive() && System.currentTimeMillis() - start < 15000) {
```

Comment:
Not part of this issue and not critical, but I hope we've fixed this in newer versions to use nanos.